x-pack/docs/en/security/authorization/built-in-roles.asciidoc

```diff
@@ -127,7 +135,10 @@ Grants the minimum privileges required for any user of {monitoring} other than t
 required to use {kib}. This role grants access to the monitoring indices and grants
 privileges necessary for reading basic cluster information. This role also includes
 all {kibana-ref}/kibana-privileges.html[Kibana privileges] for the {stack-monitor-features}.
-Monitoring users should also be assigned the `kibana_user` role.
+Monitoring users should also be assigned the `kibana_admin` role, or another role
+with access to Monitoring
+{kibana-ref}/kibana-privileges.html#kibana-feature-privileges[feature privilege]
+in {kib}
```

Comment:
^^ @legrego Is this right?

Reply:
Not quite. The `monitoring_user` rule will grant the appropriate feature privileges for the Stack Monitoring application, but the missing piece is a bit awkward. The `monitoring_user` role is scoped to the `kibana-*` application, which is to say, all kibana tenants. The user needs a role which grants any access to their kibana tenant (usually `kibana-.kibana`), otherwise they won't be able to access Kibana itself.

Rather than saying all of that, what do you think about something like:

> Monitoring users should also be assigned the `kibana_admin` role, or another role
with access to the {kib} instance.